### PR TITLE
Added format before commit

### PR DESCRIPTION
**Pull request by suyashvsingh**
It references issue #27 

I have fixed the merge conflict.  Before commiting the code locally, the prettier will format the files.

### CODE REVIEW

**Comment by glaslos** [comment]
Hm, can you prevent the json file to be changed?

**Comment by suyashvsingh** [comment]
Done, I have removed json from the lint-staged key in the package.json
`"lint-staged": {
        "**/*.{js,jsx,ts,tsx,html,css,svelte,md}": [
            "npx prettier --write"
        ]
    }`

**Comment by glaslos** [comment]
You are still committing a massive change to the package-lock.json file

**Comment by suyashvsingh** [comment]
@glaslos I have successfully resolved that issue.

**Comment by glaslos** [comment]
This PRs conflicts need to be resolved.

File: .github/workflows/go.yml
Comment (glaslos) [inline .github/workflows/go.yml:1]: I feel we should exclude `yml` files from prettier as well

File: package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
     "dev": "rollup -c -w",
     "start": "sirv public --no-clear",
     "check": "svelte-check --tsconfig ./tsconfig.json",
-    "test": "jest"
+    "test": "jest",
+    "prepare": "husky install"
```
Comment (glaslos) [inline package.json:11]: I don't agree with having this as a command. Wouldn't you get `husky` by doing the usual `npm install`?

File: .husky/pre-commit [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```
Comment (glaslos) [inline .husky/pre-commit:4]: why not run prettier directly?

File: package-lock.json
Comment (glaslos) [inline package-lock.json:1]: I feel like this has way too many changes? What happened here?

File: src/App.svelte [hunk 3]
```diff
@@ -17,6 +17,8 @@
         isLoggedIn = status;
     });
 
+    const ws = new WebSocket(`wss://${location.host}/subscribe`);
```
Comment (glaslos) [inline src/App.svelte:20]: Why is this here?

**Review by glaslos — CHANGES_REQUESTED** [review]

File: .github/workflows/go.yml
Comment (suyashvsingh) [reply .github/workflows/go.yml:1]: OK will do the same

File: .husky/pre-commit (hunk 2 above)
Comment (suyashvsingh) [reply .husky/pre-commit:4]: Running prettier directly was causing the formatted code to be commited, but after the commit the unformatted code came back after the commit

File: package-lock.json
Comment (suyashvsingh) [reply package-lock.json:1]: Hmm, I think I will have to start afresh to see why is this happening

**Comment by suyashvsingh** [comment]
I will create a new PR, then I think package-json issue will get solved.

File: package.json (hunk 1 above)
Comment (suyashvsingh) [reply package.json:11]: This command came after running npx husky-init command

**Comment by suyashvsingh** [comment]
I have found that the two devDepencies, husky and lint-staged have a lot of dependencies themselves, so the package-lock.json is having many additions.

File: src/App.svelte (hunk 3 above)
Comment (suyashvsingh) [reply src/App.svelte:20]: I might have pasted a other code, sorry for that
